index.html

```diff
@@ -5289,7 +5289,7 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
 			<rdef>menu</rdef>
 			<div class="role-description">
 				<p>A type of <a>widget</a> that offers a list of choices to the user.</p>
-				<p>A menu is often a list of common actions or functions that the user can invoke. The <code>menu</code> <a>role</a> is appropriate when a list of menu items is presented in a manner similar to a menu on a desktop application.</p>
+				<p>A menu is often presented as a popup list of items, representing common actions or functions a user can invoke, related to the item that the user interacted with to reveal the menu. The <code>menu</code> <a>role</a> is appropriate when a list of menu items is presented in a manner similar to a menu on a desktop application. For instance, a menu could be used to represent a context menu for its invoking element, or it would be used to render sub-menus for items of a <rref>menubar</rref> or another <code>menu</code> popup.</p>
```

Comment:
```suggestion
				<p>A menu is a popup list of items that represent common actions or functions a user can invoke. The menu and list of <code>menuitems</code> are almost always directly related to the item that the user interacted with to reveal the menu. The <code>menu</code> <a>role</a> is appropriate when a list of menu items is presented in a manner similar to a menu on a desktop application. For instance, a menu could be used to represent a context menu for its invoking element, or it would be used to render sub-menus for items of a <rref>menubar</rref> or another <code>menu</code> popup.</p>
```

I mainly think the first sentence could be split in two, that parenthetical makes it a bit hard to read, so here is my attempt! But I don't have a strong preference, if you prefer your sentence, so I'm otherwise approving.

Also I would remove the word "common" from the first sentence because it seems to me unnecessary.

Reply:
thanks @spectranaut 

pulling your change in.  I had just kept 'common' there since it was in the original text.  but yeh, unless someone thinks otherwise, i don't see the downside of removing it.